scripts/aem.js

```diff
@@ -627,6 +642,12 @@ function decorateBlock(block) {
     const section = block.closest('.section');
     if (section) section.classList.add(`${shortBlockName}-container`);
   }
+  // extract the quicklinks details if present
```

Comment:
On similar lines a previous comment, may be refactor this to a separate `decorateQuickLinks` function.

Reply:
done, moved it into a new decorator specifically for quick links as suggested